### PR TITLE
how to contribute to phpmyadmin added

### PR DESCRIPTION
**Pull request by manishbisht**

### CODE REVIEW

**Comment by Srijancse-zz** [comment]
Looks good to me! @tapasweni-pathak merge it?

File: phpyadmin.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+`git clone https://github.com/<yourgithubusername>/phpmyadmin.git`
+
+Additionally you need to install dependencies using the Composer tool:
+
+`composer update`
+
+Now Open your web browser and navigate to `http://locahost/<phpmyadmin-folder>` or `http://server-ip-address/<phpmyadmin-folder>`.
+
+![](https://media.giphy.com/media/7rj2ZgttvgomY/giphy.gif)
```
Comment (nikhita) [inline phpyadmin.md]: Nice GIF addition! :slightly_smiling_face: 

LGTM.

**Comment by tapaswenipathak** [comment]
@manishbisht These instructions will work on any operating system? If you are not sure, can you add the details of your machine? This most of the time creates issue when people try to install on their machine.

Also can you change the name of the file as orgname_projectname_yourname.md. It will be easy to search for any, in future.

Feel free to add anything else that helped you while you were starting, that's what makes things easier for others.

**Comment by manishbisht** [comment]
@tapasweni-pathak Updated the PR.

**Comment by tapaswenipathak** [comment]
@Srijancse @nikhita Thanks for the review! Feel free to merge if you feel no further improvements can be done in the pull requests.

**Pull request merged**